luarules/gadgets/unit_timeslow.lua

```diff
@@ -0,0 +1,340 @@
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+function gadget:GetInfo()
+   return {
+      name      = "Time slow v2",
+      desc      = "Time slow Weapon",
```

Comment:
Can this be more descriptive?

Reply:
Edited it a little, unsure where it displays or how verbose they're meant to be